lib/cpp/src/thrift/concurrency/ThreadManager.cpp

```diff
@@ -339,103 +357,88 @@ void ThreadManager::Impl::addWorker(size_t value) {
     idMap_.insert(std::pair<const Thread::id_t, shared_ptr<Thread> >((*ix)->getId(), *ix));
```

Comment:
Removing seems quite safe to me too.
But if you intend this for 0.10.0, it may not be the best timing.
Thought ?
Otherwise it looks good to me, thanks.